sci-libs/minc/minc-2.2.00.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Medical Imaging file format and Toolbox"
+HOMEPAGE="http://packages.bic.mni.mcgill.ca/tgz/${P}.tar.gz"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="BSD"
+
+COMMON_DEP="x11-libs/libXmu
+	x11-libs/libXi"
+
+DEPEND="sys-devel/flex
+	sys-devel/autoconf
```

Comment:
it is listed as a dep [here](https://en.wikibooks.org/wiki/MINC/Tutorials/SourceInstall).